media_kit_test/lib/common/widgets.dart

```diff
@@ -1,8 +1,12 @@
 import 'dart:async';
```

Comment:
Many widgets are no longer necessary now that the new controls have been implemented.

Reply:
I was also considering to remove them.

I thought they may act like a good "code sample" for people e.g. the `SeekBar`.